### PR TITLE
Deep-link Side-panel Updates

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Hi @ChetanSharma-msft @Prithvi-MSFT 

Raied PR for Deeplink Sidepanel Updates

Please review and provide feedback.

Thanks.

### CODE REVIEW

File: samples/tab-deeplink/nodejs/README.md [hunk 1]
```diff
@@ -125,6 +125,17 @@ Enter text in the emulator.  The text will be echoed back by the bot.
 
 ![Meeting stage view](Images/MeetingStageView.png)
 
+6. **Deeplink to meeting side panel:**
+
+`@mention` bot in meeting chat to get adaptive card.
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/nodejs/README.md:130]: `@mention` bot in meeting chat to get an adaptive card.

File: samples/tab-deeplink/nodejs/README.md [hunk 2]
```diff
@@ -125,6 +125,17 @@ Enter text in the emulator.  The text will be echoed back by the bot.
...
+
+![Bot-Adaptive-Card](DeepLinkBot/images/BotAdaptiveCard.png)
+
+Click on `Side Panel Deeplink` which will redirect to meeting side panel.
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/nodejs/README.md:134]: If suppose meeting is not going on, what will happen?

Kindly add that flow as well.

File: samples/tab-deeplink/nodejs/pages/DeepLinkTabHelper.js [hunk 3]
```diff
@@ -10,16 +10,27 @@ GetDeepLinkTabChannel = (subEntityId, ID, Desc, channelId,AppID,EntityID)=>{
...
-   let taskContext = encodeURI(`{"subEntityId": "${subEntityId}"}`);
+   let taskContext =  encodeURI(`{"subEntityId": "${subEntityId}"}`);
      return {
       linkUrl:"https://teams.microsoft.com/l/entity/"+AppID+"/"+process.env.Tab_Entity_Id +"?context=" + taskContext,
       ID:ID,
       TaskText:Desc
      }    
 }
 
+GetDeepLinkToMeetingSidePanel = (ID, Desc,AppID,baseUrl,chatId,contextType) =>{
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/nodejs/pages/DeepLinkTabHelper.js:21]: Add method summary as well including proper link spacing before return statement.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

**Comment by ChetanSharma-msft** [comment]
@Mohammed-MSFT - Kindly run build pipeline and add screenshot.

File: samples/tab-deeplink/nodejs/README.md (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/nodejs/README.md]: added

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5353218090

**Pull request merged**